docs/custom-ca.md

```diff
@@ -0,0 +1,15 @@
+# Install using custom CA certificate
+
+k0s generates all needed certificates automatically in the `<data-dir>/pki` directory (`/var/lib/k0s/pki`, by default).  
+
+But sometimes there is a need to have the CA certificate in advance.
+To make it work, just put `ca.key` and `ca.crt` files to the `<data-dir>/pki`:
+
+```shell
+mkdir -p /var/lib/k0s/pki
+cd /var/lib/k0s/pki
+openssl genrsa -out ca.key 2048
+openssl req -x509 -new -nodes -key ca.key -sha256 -days 365 -out ca.crt -subj "/CN=Custom CA"
+```
+
+Then you can [install k0s as usual](./install.md).
```

Comment:
Maybe add a chapter on how to create and use pre-shared tokens too

Reply:
Done